packages/web-react/src/components/Alert/Alert.tsx

```diff
@@ -3,19 +3,22 @@ import classNames from 'classnames';
 import { useAlertStyleProps } from './useAlertStyleProps';
 import { SpiritAlertProps } from '../../types';
 import { useStyleProps } from '../../hooks/styleProps';
+import { Icon } from '../Icon';
 
 const defaultProps = {
   color: 'success',
+  isCentered: false,
 };
 
 export const Alert = <T extends ElementType = 'div'>(props: SpiritAlertProps<T>): JSX.Element => {
   const { elementType: ElementTag = 'div', children, ...restProps } = props;
```

Comment:
```suggestion
  const { elementType: ElementTag = 'div', children, iconName, ...restProps } = props;
```

And then add logic using `useAlertIcon(iconName)` or some kind of logic in the body.

Reply:
Updated, check it :)